libr/cons/dietline.c

```diff
@@ -471,12 +472,15 @@ R_API void r_line_autocomplete() {
 				memmove (p + tt, t, len_t);
 			}
 			memcpy (p, argv[0], largv0);
-			p[largv0] = ' ';
-			if (!len_t) {
-				p[largv0 + 1] = '\0';
+
+			if (p[largv0 - 1] != '/') {
+				p[largv0] = ' ';
+				if (!len_t) {
+					p[largv0 + 1] = '\0';
+				}
 			}
-			I.buffer.length = strlen (I.buffer.data);
-			I.buffer.index = (p - I.buffer.data) + largv0 + 1;
+		I.buffer.length = strlen (I.buffer.data);
```

Comment:
wrong indent

Reply:
Sorry, still a noob with github and git. I need to get my workflow right. I gonna fix it and resubmit. Thank you. 
mrpaja